src/templates/components/healthServices/index.tsx

```diff
@@ -0,0 +1,35 @@
+import ServicesList from '../servicesList'
+import { HealthServices as FormattedHealthServices } from '@/types/formatted/healthServices'
+
+interface HealthServicesProps {
+  services: FormattedHealthServices
+  typeOfCare: string
+}
+
+function HealthServices({ services, typeOfCare }: HealthServicesProps) {
+  if (!services.length) return null
+
+  const headingsMap = {
```

Comment:
this can be outside of the component, the map doesn't need to re-created on every render

Reply:
Good catch, fixed